crates/typst-library/src/text/raw.rs

```diff
@@ -56,6 +62,38 @@ use crate::World;
 /// #raw("fn " + "main() {}", lang: "rust")
 /// ```
 ///
+/// # Styling
```

Comment:
Maybe we want to move this below the "Syntax" section because that one is a bit more basic?